cfdm/read_write/netcdf/netcdfwrite.py

```diff
@@ -10,11 +10,16 @@
 
 from .. import IOWrite
 
+from .netcdfread import NetCDFRead
+
+from ...cfdmimplementation import implementation
```

Comment:
Better to use `self.implementation`, i.e.  `effective_fields = NetCDFRead(self.implementation).read(filename)`, as this will play nicely with downstream applications (i.e. cf.python) 
```suggestion
```

Reply:
Oops, ignore my comment on the previous item saying I'll remove this in the next commit, you've covered it. Thanks.